<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editTask</class>
 <widget class="QWidget" name="editTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 10pt &quot;微软雅黑&quot;;
	background-color:rgb(255, 255, 255);
	border-radius:20px;
	border:1px solid rgba(45, 45, 45, 0.5);
}

QLabel{
	border:none;
	font: 11pt &quot;HONOR Sans Display GBK&quot;;
	background-color:rgba(0,0,0,0);
	/*font: 10pt &quot;MS Reference Sans Serif&quot;;*/
}

QLabel#taskTitle{
	font: 600 15pt &quot;Segoe UI Variable Display Semib&quot;;
}

QTextEdit{
	border-radius:10px;
	/*font: 10pt &quot;MS Reference Sans Serif&quot;;*/
	padding: 4px 0px; 
	text-align: left;
}

QLineEdit{
	border-radius:10px;
	padding: 4px 0px; 
	text-align: left;
}

QComboBox{
	border-radius:10px;
	border:1px solid rgba(45, 45, 45, 0.5);
}

QComboBox::drop-down{
	border:none;
	width:25px;
	border-top-right-radius:10px;
	border-bottom-right-radius:10px;
}

QComboBox::down-arrow{
	image:url(:/img/Chevron-down.svg);
	width:15px;
	height:15px;
}

QComboBox::drop-down:hover{
	background-color:rgba(203, 203, 203,0.5);
}

QComboBox QAbstractItemView{
	border-radius:5px;
}

QComboBox QAbstractItemView::item{
		height:30px;
}

QPushButton{
	border:none;
	border-radius:10px;
}

QPushButton:hover{
	background-color:rgba(203,203,203,0.5);
}

QPushButton#okButton{
	background-color:black;
	color:white;
}

QCheckBox{
	border:none;
	background-color:rgba(0,0,0,0);
	font: 11pt &quot;HONOR Sans Display GBK&quot;;
}</string>
  </property>
  <widget class="QWidget" name="mainwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>610</height>
    </rect>
   </property>
   <widget class="QLabel" name="taskIcon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resource/icons.qrc">:/img/Clipboard.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="taskTitle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>331</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>TASK</string>
    </property>
   </widget>
   <widget class="QLabel" name="taskName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Task Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="category">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
   <widget class="QComboBox" name="labelChoose">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="description">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="process">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="processEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>821</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>520</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ddlCalendar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#ddlCalendar{
	image:url(:/img/Icon button.svg);
	background-color:rgba(0,0,0,0);
}

QPushButton#ddlCalendar:hover{
	background-color:rgba(203,203,203,0.5);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ddlText">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ddlTimeText">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="repeatEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="Deadline">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Deadline</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="repeat">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Recurring Cycle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>200</y>
      <width>69</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Days</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameAlert">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#nameAlert{
	color:red;
}</string>
    </property>
    <property name="text">
     <string>* A name is required</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="descriptionEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>821</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>480</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#deleteButton{
	background-color:rgba(0,0,0,0);
}

QPushButton#deleteButton:hover{
	background-color:rgba(203,203,203,0.5);
}</string>
    </property>
    <property name="text">
     <string>                                </string>
    </property>
   </widget>
   <widget class="QLabel" name="deleteText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete this Task</string>
    </property>
   </widget>
   <zorder>taskIcon</zorder>
   <zorder>taskTitle</zorder>
   <zorder>taskName</zorder>
   <zorder>nameEdit</zorder>
   <zorder>category</zorder>
   <zorder>labelChoose</zorder>
   <zorder>description</zorder>
   <zorder>process</zorder>
   <zorder>processEdit</zorder>
   <zorder>okButton</zorder>
   <zorder>cancelButton</zorder>
   <zorder>ddlText</zorder>
   <zorder>ddlCalendar</zorder>
   <zorder>ddlTimeText</zorder>
   <zorder>repeatEdit</zorder>
   <zorder>Deadline</zorder>
   <zorder>repeat</zorder>
   <zorder>label</zorder>
   <zorder>nameAlert</zorder>
   <zorder>descriptionEdit</zorder>
   <zorder>deleteButton</zorder>
   <zorder>deleteText</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../resource/icons.qrc"/>
 </resources>
 <connections/>
</ui>
